src/openforms/forms/api/viewsets.py

```diff
@@ -388,6 +391,45 @@ def export(self, request, *args, **kwargs):
         response["Content-Length"] = len(response.content)
         return response
 
+    @extend_schema(
+        summary=_("Set default language for form"),
+        tags=["forms"],
+        request=None,
+        responses={status.HTTP_201_CREATED: FormSerializer},
+        parameters=[
+            UUID_OR_SLUG_PARAMETER,
+            OpenApiParameter(
+                name="Location",
+                type=OpenApiTypes.URI,
+                location=OpenApiParameter.HEADER,
+                description="URL of the created resource",
+                response=[status.HTTP_201_CREATED],
+            ),
+        ],
+    )
+    @transaction.atomic
+    @action(
+        detail=True,
+        methods=["post"],
+        authentication_classes=(),
+        permission_classes=(),
```

Comment:
This endpoint probably shouldn't require the user to be authenticated/have permissions?

Reply:
indeed not, because they typically wouldn't have started a submission yet :+1: